app/models/approval.rb

```diff
@@ -16,6 +16,7 @@ class Approval < ActiveRecord::Base
 
   scope :individual, -> { where(type: 'Approvals::Individual') }
 
+  validates :proposal, presence: true, allow_blank: true
```

Comment:
@cmc333333 Will all Approvals be associated with a Proposal?

Also, I think `presence: true, allow_blank: true` is basically canceling itself out :wink: